File: app/src/main/java/com/zwstudio/logicpuzzlesandroid/puzzles/taparow/domain/TapARowHintObject.java
package com.zwstudio.logicpuzzlesandroid.puzzles.taparow.domain;

import com.zwstudio.logicpuzzlesandroid.home.domain.HintState;

public class TapARowHintObject extends TapARowObject {
    public HintState state = HintState.Normal;
    public String objTypeAsString() {
        return "hint";
    }
}

File: app/src/main/java/com/zwstudio/logicpuzzlesandroid/puzzles/taparow/domain/TapARowHintObject.kt
package com.zwstudio.logicpuzzlesandroid.puzzles.taparow.domain

import com.zwstudio.logicpuzzlesandroid.home.domain.HintState

class TapARowHintObject : TapARowObject() {
    var state = HintState.Normal
    override fun objTypeAsString(): String {
        return "hint"
    }
}